src/client/src/hooks/useRole.tsx

```diff
@@ -15,8 +15,9 @@ export function useRole() {
             const email = user.primaryEmailAddress?.emailAddress;
             if (!email) return;
 
+            const host = window.location.hostname;
             const response = await axios.get(
```

Comment:
Could you explain why did you change the localhost with host?
In this case if you want to make a production ready url you can add in the `.env.example`a variable called `BASE_URL=http://localhost:3000/`which in our local environment should be localhost.